redisgears_core/Cargo.toml

```diff
@@ -22,12 +22,8 @@ lazy_static = "1"
 log = "0.4"
 byte-unit = "4"
 serde_json = "1"
-# DO NOT CHANGE to avoid security issues. This exact version
-# specification is required for the second level dependencies
-# to use exactly this version too, so that the dependencies of
-# this project will not be able to compromise it.
-serde = { version = "=1.0.171", features = ["derive"] }
-serde_derive = "=1.0.171"
+serde = { version = "1", features = ["derive"] }
```

Comment:
Maybe we can put the serde on the workspace cargo.toml like I did with redismodule-rs?